controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -320,6 +320,58 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) (*corev
 		// Else: nothing for eBPF at the moment
 	}
 
+	outputFields := []api.OutputField{
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+		},
+		{
+			Name:      "Bytes",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+		},
+		{
+			Name:      "Packets",
+			Operation: "sum",
+			SplitAB:   true,
+		},
+		{
+			Name:      "numFlowLogs",
+			Operation: "count",
+		},
+		{
+			Name:      "TimeFlowStartMs",
+			Operation: "min",
+		},
+		{
+			Name:      "TimeFlowEndMs",
+			Operation: "max",
+		},
+		{
+			Name:      "FlowDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "IfDirection",
+			Operation: "first",
+		},
+		{
+			Name:      "AgentIP",
+			Operation: "first",
+		},
+	}
+
+	if b.desired.Agent.EBPF.EnableFlowRTT {
+		outputFields = append(outputFields, api.OutputField{
+			Name:      "FlowRTT",
+			Operation: "last",
+		})
```

Comment:
I wonder what we want to highlight here :thinking: 
Maybe the average and max value would be more interesting here

FLP doesn't implement average yet:
https://github.com/netobserv/flowlogs-pipeline/blob/main/pkg/pipeline/extract/conntrack/aggregator.go#L57

For max you can use:
```suggestion
		outputFields = append(outputFields, api.OutputField{
			Name:      "MaxFlowRTT",
			Operation: "max",
		})
```

Reply:
Reopening this since the field has been renamed and Input was not set:
https://github.com/netobserv/network-observability-operator/pull/394/commits/9960f9eec94b085c60d488069423bb656e37edb5